### ISSUE TITLE
Provide an icon font

### ISSUE THREAD

**Issue by nt1m** [FirefoxUX/icons#48]
It's handy for the web if we could generate an icon font from the SVGs.

There are some tools to do that, like fontello: http://fontello.com/

**Comment by nt1m** [comment]
The challenges this repo have to create an icon font are
- there are duplicate icons (themes and customize for example)
- the size suffix will cause the generated CSS to also include the suffix

**Comment by aminalhazwani** [comment]
I am not really sure about icons font, there bring to the table a lot of drawbacks, I can provide more details but this talk is a very good summary https://speakerdeck.com/ninjanails/death-to-icon-fonts

**Comment by brassy-** [comment]
What @aminalhazwani said. @nt1m I'd like to close this issue if it's ok for you.

**Comment by nt1m** [comment]
@brassy- @aminalhazwani I agree with a lot of arguments on the slides, but where the icon fonts really win over inline SVG is webpage performance (creating SVG markup costs).

I don't think it's extremely essential though.

**Comment by aminalhazwani** [comment]
Hey @nt1m feel free to give it a take on the /photon-icons repo https://github.com/FirefoxUX/photon-icons 🙂
